crates/macro/ui-tests/main-infallible.stderr

```diff
@@ -3,3 +3,20 @@ error: the main function has to be called main
    |
 10 | fn fail() {}
    |    ^^^^
+
+warning: unreachable expression
+ --> ui-tests/main-infallible.rs:4:1
+  |
+4 | #[wasm_bindgen(main)]
+  | ^^^^^^^^^^^^^^^^^^^^^
+  | |
+  | unreachable expression
+  | any code following this expression is unreachable
+  |
+note: this expression has type `Infallible`, which is uninhabited
+ --> ui-tests/main-infallible.rs:4:1
+  |
+4 | #[wasm_bindgen(main)]
+  | ^^^^^^^^^^^^^^^^^^^^^
+  = note: `#[warn(unreachable_code)]` on by default
+  = note: this warning originates in the attribute macro `wasm_bindgen` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Comment:
Fun. That's a bug. We should avoid emitting this warning somehow.
Not related to this PR.